src/utils.ts

```diff
@@ -0,0 +1,107 @@
+import { html, TemplateResult } from 'lit-html'
```

Comment:
That is fine. Created folder `utils` containing `I18N.ts` and `HelperFunctions.ts`.

I'm not that well versed in TypeScript, and though putting everything in it's own class was unnecessary for the helper functions

Reply:
Yes, it's not necessary but it helps to keep some consistency, it could be consider some a bit subjective but if there's not a reason to not do it, I'll rather go in that direction